### ISSUE TITLE
just random question

### ISSUE THREAD

**Issue by ganesh1997oli** [yoodamchoi/artTokenization_ink#1]
hey, I saw `readme` file of the repo which is really awesome idea for nft authentication and ownership. Are you working on this project?

**Comment by yoodamchoi** [comment]
Hi Ganesh oli.Thanks for asking. I’m trying to working on it. Right now, I’ve done contract and testing work but didn’t deploy yet.Is there reason why you’re asking? Just curious.On Mar 28, 2023, at 8:23 AM, Ganesh oli ***@***.***> wrote:﻿
hey, I saw readme file of the repo which is really awesome idea for nft authentication and ownership. Are you working on this project?

**Comment by ganesh1997oli** [comment]
yeah saw your project and like the idea and I thought may be I could contribute something like writing contract or anything. BTW thanks for quick reply!

**Comment by yoodamchoi** [comment]
Oh that’s good though. What do you do? On Mar 28, 2023, at 12:13 PM, Ganesh oli ***@***.***> wrote:﻿
yeah saw your project and like the idea and I thought may be I could contribute something like writing contract or anything. BTW thanks for quick reply!

**Comment by ganesh1997oli** [comment]
I write smart contract and learning substrate chain

**Comment by ganesh1997oli** [comment]
and you?

**Comment by yoodamchoi** [comment]
I’m studying blockchain development and also finding some blockchain area to work. So I was trying to do some project to appeal myself.On Mar 28, 2023, at 12:23 PM, Ganesh oli ***@***.***> wrote:﻿
and you?

**Comment by ganesh1997oli** [comment]
that's nice if you are comfortable may be we can work together to build something!

**Comment by yoodamchoi** [comment]
That’s great idea. These days I’m free most time and mostly just making projects. Where do you currently live?On Mar 28, 2023, at 9:35 PM, Ganesh oli ***@***.***> wrote:﻿
that's nice if you are comfortable may be we can work together to build something!

**Comment by ganesh1997oli** [comment]
I am from Nepal

**Comment by ganesh1997oli** [comment]
are you on discord? If you are then you can add me and message me on `go123#9432` username
